code/modules/mob/living/simple_animal/friendly/lizard.dm

```diff
@@ -19,3 +20,20 @@
 	pass_flags = PASSTABLE | PASSMOB
 	mob_size = MOB_SIZE_SMALL
 	gold_core_spawnable = 2
+	environment_smash = 0
+
+/mob/living/simple_animal/hostile/lizard/CanAttack(atom/the_target)//Can we actually attack a possible target?
+	if(see_invisible < the_target.invisibility)//Target's invisible to us, forget it
+		return 0
+	if(istype(the_target,/mob/living/simple_animal/butterfly) || istype(the_target,/mob/living/simple_animal/cockroach))

```

Comment:
I could. This reminds me of a bug I found while testing where player lizards could eat EVERYTHING they clicked on. As you can see lower in the file, there's now a check there too.